hyperspy/Release.py

```diff
@@ -25,7 +25,7 @@
 # When running setup.py the ".dev" string will be replaced (if possible)
 # by the output of "git describe" if git is available or the git
 # hash if .git is present.
-version = "1.6.0.dev"
+version = "1.7.0.dev"
```

Comment:
Don't forget you're based on the wrong branch here.

Reply:
Thanks, I had changed it on github but not rebased. Dealt with in 72703cb

Reply:
Pretty certain this file shouldn't be showing up at all in the "changed" list in this PR?